### ISSUE TITLE
(iOS SDK 8) Attempts to modify the starred playlist have no effect.

### ISSUE THREAD

**Issue by trufflin** [spotify/ios-sdk#356]
I am retrieving my own starred playlist using the following (not deprecated) method: 

[SPTPlaylistSnapshot requestStarredListForUserWithSession:callback:]

I receive a valid playlist snapshot in return. I have tried calling both the following methods on this object, in an attempt to modify the starred playlist: 

[SPTPlaylistSnapshot replaceTracksInPlaylist:withAccessToken:callback:]
[SPTPlaylistSnapshot addTracksToPlaylist:withAccessToken:callback:]

Both methods will execute and return a NIL error to the callback block; however, the starred playlist will not be modified. I have confirmed this by viewing my starred playlist in the official Spotify Mac client. 

I have been using the same code flow to edit my other playlists, which has been working fine. As far as I can tell, it is not an issue with authenticated scopes, etc. (again, the request to edit is not returning any error.) 

Thank you.

**Comment by thebradbain** [comment]
I am having this exact same issue, has there been any headway?

**Comment by erdtman** [comment]
Sorry for not getting back earlier. In an attempt to get a better overview of the issues we are cleaning out some of the older issues. If this issue is still a problem in the latest SDK please reopen this issue or open a new one.

**Comment by dsuper66** [comment]
I have found that  replaceTracksInPlaylist does not work (but does not error) if you supply it with more than 100 tracks, i.e., the page limit. I am trying to use it to move a track within the playlist, it works fine if there are less than 100 tracks. 

As an alternative I tried a removeTracksWithPositionsFromPlaylist followed by an addTracksWithPositionToPlaylist... the remove works, but the add does not (regardless of the number of tracks in the playlist)... they seem to be quite different, i.e., the argument name for the token is different, and the remove uses a position and track dictionary (which works), while the add just uses and index (and doesn't work).
